luaconf.h

```diff
@@ -872,9 +883,16 @@ static inline int time(void *p)
 
 /* stdlib.h */
 #include <linux/slab.h>
-#define free(a) 			kfree(a)
+#define free(a) 		kfree
```

Comment:
`kfree` -> `kfree(a)`

Reply:
done.. please take a look, I did `free(a)` -> `free`, instead. do you seen any issue?

Reply:
yep, should work, but function style macro looks a little bit safer because it's applied less widely.

but as far as I can see we already have this https://github.com/luainkernel/lua/commit/2462d90b503c9b4f7e0d85791903fc075a10e49a

Reply:
@mihalicyn you're absolutely right.. I've converted all macros I modified/added to that style.. we should change all macros later on.. please take a look.. thanks!